src/Model.js

```diff
@@ -811,7 +811,10 @@ export class Model {
             if (params.exists == null) {
                 let field = this.block.fields[this.createdField] || this.table
                 let when = (field.isoDates) ? now.toISOString() : now.getTime()
-                params.set = { [this.createdField]: `if_not_exists(\${${this.createdField}}, {${when}})` }
+                params.set = {
+                    ...params.set,
+                    [this.createdField]: `if_not_exists(\${${this.createdField}}, {${when}})`
+                }
```

Comment:
Could this not be more easily written as:

params.set = params.set || {}
params.set[this.createdField] = `if_not_exists(\${${this.createdField}}, {${when}})`

Reply:
yeah, looks better indeed, will apply it